recipes/opus/all/conandata.yml

```
@@ -1,4 +1,7 @@
 sources:
+  "1.5.1":
+    url: "https://downloads.xiph.org/releases/opus/opus-1.5.1.tar.gz"
+    sha256: "b84610959b8d417b611aa12a22565e0a3732097c6389d19098d844543e340f85"
```

```suggestion
  "1.5.2":
    url:
      - "https://ftp.osuosl.org/pub/xiph/releases/opus/opus-1.5.2.tar.gz"
      - "https://downloads.xiph.org/releases/opus/opus-1.5.2.tar.gz"
    sha256: "65c1d2f78b9f2fb20082c38cbe47c951ad5839345876e46941612ee87f9a7ce1"
```

The downloads.xiph.org seems to redirect to mirrors. So let's try to use one of them as the preferred download.
For older versions we also have used their GitHub downloads, but they did not upload them there for 1.5x. And we also used a Mozilla mirror, but they don't seem to have these versions yet either.

Checksum is taken from https://opus-codec.org/release/stable/2024/04/12/libopus-1_5_2.html

And let's use 1.5.2 instead of 1.5.1 at this point

---

I think it will be better to use the gitlab official repo source code URL:

https://gitlab.xiph.org/xiph/opus/-/archive/v1.5.2/opus-v1.5.2.tar.gz

```suggestion
  "1.5.2":
    url: "https://gitlab.xiph.org/xiph/opus/-/archive/v1.5.2/opus-v1.5.2.tar.gz"
    sha256: "b84610959b8d417b611aa12a22565e0a3732097c6389d19098d844543e340f85"
```